proxy/http/remap/UrlRewrite.cc

```diff
@@ -769,7 +769,7 @@ UrlRewrite::TableInsert(std::unique_ptr<URLTable> &h_table, url_mapping *mapping
     h_table->emplace(src_host, ht_contents);
   }
   if (!ht_contents->Insert(mapping)) {
-    Warning("Could not insert new mapping");
+    Warning("Could not insert new mapping: duplicated entry exists.");
```

Comment:
It's not immediately obvious here that the only reason that `Insert` would return false is that there is a duplicate. Looking at [the code](https://github.com/apache/trafficserver/blob/f1ece8382a54fcbc7e3416e427b26d25b3177986/include/tscore/Trie.h#L169), as I'm sure you did, that seems to be the case. Let's add a comment assuring the reader of the code that this is true:

```
// Trie::Insert only fails due to an attempt to add a duplicate entry.
```

Reply:
Yeah this can be helpful. Added it.